test_scripts/Resumption/InteriorVehicleData/001_InteriorVD_resumption_one_moduleType_without_moduleId_unexpected_disconnect.lua

```diff
@@ -2,7 +2,7 @@
 -- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
 --
 -- Description: Successful resuming of interior vehicle data after transport disconnect
---  in case GetInteriorVehicleData was requested with single module and without moduleId
+--  in case app was subscribed on one module without moduleId
```

Comment:
Propose to update to `subscribed on` to `subscribed to`

Reply:
Done (see 3f22932 )